### PR TITLE
jamoviStats.jamovi version 2.2.5.0

### PR DESCRIPTION
**Pull request by torculus**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/43874)

### CODE REVIEW

File: manifests/j/jamoviStats/jamovi/2.2.5.0/jamoviStats.jamovi.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+PackageIdentifier: jamoviStats.jamovi
+PackageVersion: 2.2.5.0
+PackageLocale: en-US
+Publisher: jamoviStats
+PublisherUrl: https://www.jamovi.org/
+PublisherSupportUrl: https://www.jamovi.org/user-manual.html
+PrivacyUrl: https://forum.jamovi.org/ucp.php?mode=privacy
+Author: Jonathon Love, Damian Dropmann, Ravi Selker
+PackageName: jamovi 2.2.5.0
```
Comment (ItzLevvie) [inline manifests/j/jamoviStats/jamovi/2.2.5.0/jamoviStats.jamovi.locale.en-US.yaml:12]:
```suggestion
PackageName: jamovi
```

File: manifests/j/jamoviStats/jamovi/2.2.5.0/jamoviStats.jamovi.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: jamoviStats.jamovi
+PackageVersion: 2.2.5.0
+PackageLocale: en-US
+Publisher: jamoviStats
```
Comment (ItzLevvie) [inline manifests/j/jamoviStats/jamovi/2.2.5.0/jamoviStats.jamovi.locale.en-US.yaml:7]:
```suggestion
Publisher: The jamovi Project
```

File: manifests/j/jamoviStats/jamovi/2.2.5.0/jamoviStats.jamovi.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: jamoviStats.jamovi
+PackageVersion: 2.2.5.0
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://www.jamovi.org/downloads/jamovi-2.2.5.0-win64.exe
+  InstallerSha256: 4EF1B091F41D8F63803B74717017EEC3431CC845ABAB62A4F23C19A0D41307A7
```
Comment (ItzLevvie) [inline manifests/j/jamoviStats/jamovi/2.2.5.0/jamoviStats.jamovi.installer.yaml:10]:
```suggestion
  InstallerSha256: 4EF1B091F41D8F63803B74717017EEC3431CC845ABAB62A4F23C19A0D41307A7
  Scope: machine
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Looks like there are weird BIN files in this PR:
image

**Review by ItzLevvie — APPROVED** [review]

**Comment by torculus** [comment]
> Looks like there are weird BIN files in this PR: image

Sorry, this is what happens when I try to get fancy with `vim`. I squashed my commits and re-submitted

**Pull request merged**